frontends/election-manager/src/components/zero_report_print_button.tsx

```diff
@@ -36,29 +36,44 @@ export function ZeroReportPrintButton(): JSX.Element {
   const zeroReportTitle =
     'Pre-Election Unofficial Full Election Tally Report (Zero Report)';
 
-  function logZeroReportPrintSuccess() {
+  function disablePrinting() {
+    document.getElementById('zero-report')?.classList.add('no-print');
+  }
+
+  function enablePrinting() {
+    document.getElementById('zero-report')?.classList.remove('no-print');
+  }
+
+  useEffect(() => {
```

Comment:
Using useEffect to disable printing immediately after render seemed like the simplest to disable printing without complicating the `PrintableArea` component further, although that is an option if we want to avoid `useEffect`.

Reply:
I'd rather add the appropriate behavior to `PrintableArea` rather than spreading the responsibility to other components like this and `FullTestDeckTallyReport`. We don't need to drop down to raw DOM for this.